website/src/assets/site.css

```
@@ -5614,11 +5666,37 @@ html:has(> body.docs-body)::-webkit-scrollbar-corner,
+    linear-gradient(90deg, rgba(59,130,246,calc(var(--stage2-lock-route-progress, 0) * 0.055)), rgba(59,130,246,0) 38%),
+    linear-gradient(180deg, rgba(255,255,255,1), rgba(247,248,251,0.98));
+  box-shadow:
+    0 calc(12px - (var(--stage2-lock-card-progress, 0) * 2px)) 0 rgba(9,9,11,0.1),
+    0 0 0 1px rgba(39,39,42,calc(0.16 - (var(--stage2-lock-card-progress, 0) * 0.03))),
+    inset 0 0 0 1px rgba(255,255,255,0.88),
+    inset 0 -1px 0 rgba(39,39,42,0.08);
+  opacity: max(
+    var(--stage2-lock-card-progress, 0),
+    calc(
+      var(--stage2-model-detail-opacity, 1)
+      * (1 - (var(--stage2-collapse-progress, 0) * 0.72))
```

Stylelint (17.13.0)

[error] Unexpected newline before "*" (scss/operator-no-newline-before)

(scss/operator-no-newline-before)